jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+GIT_REPO="https://github.com/obnoxxx/samba-build"
+SAMBA_BRANCH='master'
+CENTOS_VERSION='7'
+CENTOS_ARCH='x86_64'
+RESULT_BASE="/tmp/samba-build/rpms"
+RESULT_DIR="${RESULT_BASE}/${SAMBA_BRANCH}/${CENTOS_VERSION}/${CENTOS_ARCH}"
+REPO_NAME="samba-nightly-${SAMBA_BRANCH}"
+REPO_FILE="${RESULT_BASE}/${REPO_NAME}.repo"
+
+artifact()
+{
+    [ -e ~/rsync.passwd ] || return 0
+    rsync -av --password-file ~/rsync.passwd ${@} gluster@artifacts.ci.centos.org::gluster/nightly-samba/
+}
+
+# if anything fails, we'll abort
+set -e
+
+# install basic dependencies for building the tarball and srpm
+yum -y install git make rpm-build mock ansible createrepo_c
+# gluster repositories contain additional -devel packages
+#yum -y install centos-release-storage-common centos-release-gluster
+
+git clone "${GIT_REPO}"
```

Comment:
You may want to clone without the full history (downloading in a cloud environment may cost extra). `git clone --depth=1` does that for you.

Reply:
Good point! Updated..